worldengine/draw.py

```diff
@@ -277,74 +275,42 @@ def get_biome_color_based_on_elevation(world, elev, x, y):
-                 numpy.random.randint(-NOISE_RANGE, NOISE_RANGE))
-
-        ####### Case 1 - elevation is very high ########
-        if elev > HIGH_MOUNTAIN_ELEV:     
-            # Modify the noise to make the area slightly brighter to simulate snow-topped mountains.
-            noise = add_colors(noise, HIGH_MOUNTAIN_NOISE_MODIFIER)
-            # Average the biome's color with the MOUNTAIN_COLOR to tint the terrain
-            biome_color = average_colors(biome_color, MOUNTAIN_COLOR)
-
-        ####### Case 2 - elevation is high ########
-        elif elev > MOUNTAIN_ELEV:   
-            # Modify the noise to make this tile slightly darker, especially draining the green
-            noise = add_colors(noise, MOUNTAIN_NOISE_MODIFIER)
-            # Average the biome's color with the MOUNTAIN_COLOR to tint the terrain
-            biome_color = average_colors(biome_color, MOUNTAIN_COLOR)
-
-        ####### Case 3 - elevation is somewhat high ########
-        elif elev > HIGH_HILL_ELEV:   
-            noise = add_colors(noise, HIGH_HILL_NOISE_MODIFIER)
-
-        ####### Case 4 - elevation is a little bit high ########
-        elif elev > HILL_ELEV:   
-            noise = add_colors(noise, HILL_NOISE_MODIFIER)
+        noise = rng.randint(-NOISE_RANGE, NOISE_RANGE, size=3)  # draw three random numbers at once
+
+    ####### Case 1 - elevation is very high ########
+    if world.is_high_mountain((x, y)):
+        # Modify the noise to make the area slightly brighter to simulate snow-topped mountains.
+        noise = add_colors(noise, HIGH_MOUNTAIN_NOISE_MODIFIER)
+        # Average the biome's color with the MOUNTAIN_COLOR to tint the terrain
+        biome_color = average_colors(biome_color, MOUNTAIN_COLOR)
+
+    ####### Case 2 - elevation is high ########
+    elif world.is_medium_mountain((x, y)):
+        # Modify the noise to make this tile slightly darker, especially draining the green
+        noise = add_colors(noise, MOUNTAIN_NOISE_MODIFIER)
+        # Average the biome's color with the MOUNTAIN_COLOR to tint the terrain
+        biome_color = average_colors(biome_color, MOUNTAIN_COLOR)
+
+    ####### Case 3 - elevation is somewhat high ########
+    elif world.is_low_mountain((x, y)):
+        noise = add_colors(noise, HIGH_HILL_NOISE_MODIFIER)
+
+    ####### Case 4 - elevation is a little bit high ########
+    elif world.is_hill((x, y)):
+        noise = add_colors(noise, HILL_NOISE_MODIFIER)
 
     # There is also a minor base modifier to the pixel's rgb value based on height
     modification_amount = int(elev / BASE_ELEVATION_INTENSITY_MODIFIER)
     base_elevation_modifier = (modification_amount, modification_amount, modification_amount)
 
     this_tile_color = add_colors(biome_color, noise, base_elevation_modifier)
-
     return this_tile_color
+
+
 # ----------------------
 # Draw on generic target
 # ----------------------
 
-
-class ImagePixelSetter(object):
```

Comment:
This is due to PR #163 . The functions for drawing haven't really been changed; the image-handler is now in ```image_io.py``` - a reader and a writer.